Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -567,9 +567,6 @@ class TabTrayController: UIViewController {
         // If we are exiting private mode and we have the close private tabs option selected, make sure
         // we clear out all of the private tabs
         let exitingPrivateMode = !privateMode && profile.prefs.boolForKey("settings.closePrivateTabs") ?? false
```

Comment:
Nit: use `tabManager.shouldClearPrivateTabs() ` instead of ` profile.prefs.boolForKey("settings.closePrivateTabs") ?? false`